app/views/orders/show.html.erb

```diff
@@ -25,7 +25,7 @@
                             width: 80,
                             class: "img"
                         ),
-                        photo_path(order_photo) %></td>
+                        photo_path(order_photo.photo_id) %></td>
```

Comment:
the order_photo was not working as its own object, similar to the issue we had yesterday